pandas/io/parsers.py

```diff
@@ -2174,7 +2195,14 @@ def __init__(self, f, **kwds):
 
         self.verbose = kwds['verbose']
         self.converters = kwds['converters']
-        self.dtype = kwds['dtype']
+
```

Comment:
this shouldn't be done here, rather you can do this if needed inside the parser itself